apps/dev/src/card/card-demo.component.html

```diff
@@ -4,7 +4,6 @@
     <dt-card-subtitle>Some subtitle</dt-card-subtitle>
     <dt-card-title-actions>
       <button dt-button variant="secondary">Some Action</button>
-      <button dt-button variant="secondary">Action</button>
```

Comment:
Please add the following back:
` button dt-button variant="secondary">Action</button>`